datadog_checks_base/datadog_checks/base/utils/common.py

```diff
@@ -25,7 +25,8 @@ def ensure_unicode(s):
     return s
 
 
-to_string = ensure_unicode if PY3 else ensure_bytes
+to_native_string = ensure_unicode if PY3 else ensure_bytes
+to_string = to_native_string  # For backwards compat before when this was renamed to `to_native_string`.
```

Comment:
Could you add a deprecation warning?

Reply:
You may need to convert `to_string` into a function.

Reply:
Sure!

Reply:
Added in 92a8187 along with missing tests.